rc/detection/modeline.kak

```diff
@@ -30,14 +30,16 @@ define-command -hidden modeline-parse-impl %{
                     case "${value}" in
                         unix) tr="eolformat lf";;
                         dos) tr="eolformat crlf";;
-                        *) printf %s\\n "echo -debug 'Unsupported file format: ${value}'";;
+                        *) printf 'echo -debug %s' "$(kakquote "Unsupported file format: ${value}")" \
+                               | kak -p "${kak_session}";;
```

Comment:
Would printing to stderr do the same? I'm not sure.

Reply:
When writing the message to stderr, this is what the debug buffer looks like:

```
*** This is the debug buffer, where debug info will be written ***
shell stderr: <<<
Unsupported modeline format: # foo: hey

>>>
shell stderr: <<<
Unsupported modeline format: shell stderr: <<<

>>>
```

Reply:
Oh sorry, this would have been easy to check myself. Thanks for elaborating!